docs/modules/setup/examples/k8s-single-pod/kroki-deployment.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kroki
+  labels:
+    app: kroki
+
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kroki
+  template:
+    metadata:
+      labels:
+        app: kroki
+    spec:
+      containers:
+      - name: kroki
+        image: docker.io/yuzutech/kroki:0.18.0
```

Comment:
It's a good practice to set a version but I think we should use "latest" or update this version when we release a new version. 
My concern is that someone in a few months/years will copy/paste this deployment and run an outdated version.

Reply:
Very true, wasn't thinking about that, mainly just copied & pasted from what I was using in my own cluster, haha.